grails-app/services/org/pih/warehouse/core/UserService.groovy

```diff
@@ -171,11 +171,10 @@ class UserService {
 
 
     Boolean isUserInRole(String userId, Collection roleTypes) {
-        Collection acceptedRoleTypes = RoleType.expand(roleTypes)
         User user = getUser(userId)
         return getEffectiveRoles(user).any { Role role ->
-            boolean acceptedRoleType = acceptedRoleTypes.contains(role.roleType)
-            log.info "Is role ${role.roleType} in ${acceptedRoleTypes} = ${acceptedRoleType}"
+            boolean acceptedRoleType = roleTypes.contains(role.roleType)
```

Comment:
Can you explain why you removed the expand part? That's one of the more crucial aspects of the user role validation.

Reply:
@jmiranda I have forgotten that we use specific methods to check specific roles. I've added hasRoleApprover method in order to not change this role validation.